actionpack/lib/action_controller/metal.rb

```diff
@@ -41,19 +41,19 @@ def build(action, app = nil, &block)
 
       def build_middleware(klass, args, block)
         options = args.extract_options!
-        only   = Array(options.delete(:only)).map(&:to_s)
-        except = Array(options.delete(:except)).map(&:to_s)
+        only   = Array(options.delete(:only)).map!(&:to_s)
```

Comment:
My only concern would be modifying the options passed in:

```rb
only = [:index, :create]
PostsController.fast_use(SomeMiddleware, only: only)
p only
#=> ["index", "create"]
```

Is there any risk there?

Reply:
I played around a bit more and indeed `Array(smth)` will return the original if it's already an array. Didn't realize that.